datalad/support/gitrepo.py

```diff
@@ -1308,6 +1309,9 @@ def for_each_ref_(self, fields=('objectname', 'objecttype', 'refname'),
           descending order.
         count : int, optional
           Stop iteration after the given number of matches.
+        contains : str, optional
+          Only list refs which contain the specified commit (HEAD if not
+          specified).
```

Comment:
If not specified, it's `None`, and `--contains` isn't added.  That's not the same as restricting the results to `--contains=HEAD`.

Reply:
Fixed, by removing that part from the docs (copied from Git).